file/kong_json_schema.json

```diff
@@ -444,7 +444,6 @@
         },
         "groups": {
           "items": {
-            "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Is this change related to Consumer Groups work? 🤔

Reply:
In that case: #991, let's mark it as generated to prevent folks from reviewing this.